missing_locales.py

```diff
@@ -63,20 +84,33 @@ def main():
         dest="github_path",
         help="GitHub path that contains locale folders",
     )
+    parser.add_argument(
+        "--csv",
+        required=False,
+        action="store_true",
+        default=False,
+        dest="csv_output",
+        help="Store data as output.csv",
+    )
 
     args = parser.parse_args()
 
-    missing_locales = ["Missing Locales"]
     pontoon_locales = retrieve_pontoon_locales(args.pontoon_project)
-    github_locales = retrieve_github_locales(args.github_owner, args.github_repo, args.github_path)
+    github_locales = retrieve_github_locales(
+        args.github_owner, args.github_repo, args.github_path
+    )
+
+    output = ["Missing Locales"]
+    missing_locales = list(set(github_locales) - set(pontoon_locales))
```

Comment:
This replaces your FOR cycle, taking advantage of `set()`. 

By converting a list to a set, you remove all duplicate items (because that's how sets are defined). Then you can calculate the difference between two sets, and convert it back to a list.

Reply:
👍  Thanks!